src/main/java/ch/njol/skript/expressions/ExprAnvilText.java

```diff
@@ -0,0 +1,75 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.inventory.AnvilInventory;
+import org.bukkit.inventory.Inventory;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Anvil Input Text")
+@Description("Returns the name to be applied to the repaired item.")
+@Examples({
+		"on inventory click:",
+		"\ttype of event-inventory is anvil inventory",
+		"\tif anvil input text = \"FREE OP\":",
+		"\t\tban player"})
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.11+")
+public class ExprAnvilText extends SimplePropertyExpression<Inventory, String> {
+
+	static {
+		if (Skript.methodExists(AnvilInventory.class, "getRenameText"))
+			register(ExprAnvilText.class, String.class, "anvil [inventory] [rename] input [text]", "inventories");
```

Comment:
This might just be me, but just `anvil input` seems like it could be a little unclear to me (ex: is it referring to the text input or is it referring to the item input)

Reply:
Agree, I will change that 👌